### PR TITLE
Simplify QCRAM abstract

### PR DESCRIPTION
**Pull request by LPardue**
A terse strawman proposal.

State what not why, make it more similar to HPACK abstract.

### CODE REVIEW

**Comment by LPardue** [comment]
build failed due to trailing whitespace >.<

Please judge on content.

**Comment by martinthomson** [comment]
I like the content and have fixed the whitespace.

File: draft-ietf-quic-qcram.md [hunk 1]
```diff
@@ -32,15 +32,9 @@ author:
...
-multiplexing free of head-of-line (HoL) blocking between streams. In HTTP/2,
-multiplexed streams can suffer HoL blocking due to TCP.
-
-If HTTP/2's HPACK is used for header compression, HTTP/QUIC is still vulnerable
-to HoL blocking, because of HPACK's assumption of in-order delivery.  This draft
-defines QCRAM, a variation of HPACK and mechanisms in the HTTP/QUIC mapping that
-allow the flexibility to avoid header-compression-induced HoL blocking.
+This specification defines QCRAM, a compression format for efficiently
+representing HTTP header fields, to be used in HTTP over QUIC. This is a
+variation of HPACK header compression that avoids head-of-line blocking.
```
Comment (dtikhonov) [inline draft-ietf-quic-qcram.md]: "Avoids" is an incorrect term to use: we differentiate avoidance and minimization.  In the default mode, QCRAM minimizes head-of-line blocking.

Comment (afrind) [reply draft-ietf-quic-qcram.md]: Agreed.  @dtikhonov, would you prefer 'minimizes' here, or do you think we should elaborate? 

'...that minimizes or avoids...'
'...that seeks to reduce...'

Comment (dtikhonov) [reply draft-ietf-quic-qcram.md]: As far as abstracts go, the current QCRAM abstract is short.  To keep it that way, "seeks to reduce" is probably the best way to go.  Discussion about minimization vs. avoidance does not have to be surfaced to the level of the abstract.

**Comment by LPardue** [comment]
SGTM, I'll incorporate that text into PR when at the computer next.

**Pull request merged**